wagtail/admin/views/home.py

```diff
@@ -144,3 +146,15 @@ def default(request):
     redirected to the login page.
     """
     raise Http404
+
+
+@cache_page(60 * 60 * 24 * 365)  # seconds * minutes * hours * days
+def sprite(request):
+    response = render(request, "wagtailadmin/icons/sprite.svg")
+    response["Content-Type"] = "image/svg+xml"
+    # TODO: We do not want to set cache in debug mode.
```

Comment:
I think we do, if possible? I imagine it’s going to be really annoying for Wagtail devs to see the icons flash all the time.